holidayextras/brands
# Lego WIP

#### What does this PR do? (please provide any background)
Make a start on adding branding for Lego

#### What tests does this PR have?
N/A

#### How can this be tested?
Most of this can be viewed by running the [Pattern Library] (https://github.com/holidayextras/pattern-library) locally.

1. Check out and `npm install` [Brands](https://github.com/holidayextras/brands)
2. Check out and `npm install`  [Pattern Library](https://github.com/holidayextras/pattern-library)
3. `cd */Pattern-library` and `npm link ../brands`
4. `npm start`
5. This is a test, REPEAT, this is a test

#### Screenshots / Screencast

#### What gif best describes how you feel about this work?
![legobatman](https://cloud.githubusercontent.com/assets/3897844/14821093/4c6964ea-0bc1-11e6-81e3-6d6df010ffbf.gif)

---


#### Reviewers

**Review 1** - @andrewcashmore

**Review 2** \*

**Review 3** _(optional)_

By adding a +1 you are confirming you have...
- Witnessed the work behaving as expected (this could be on the author's machine or screencast).
- Checked for coding anti-patterns.
- Checked for appropriate test coverage.
- Checked all the tests are passing.

\*  for HX this review must be completed by an SE, SA or Project Guru
